apps/stops/lib/repo.ex

```diff
@@ -160,14 +160,9 @@ defmodule Stops.Repo do
     |> Keyword.get(:connections)
     |> get_stop_connections(stop_id)
     |> Enum.map(icon_fn)
-    |> Enum.map(&consolidate_silver_line_route_feature/1)
     |> Enum.uniq()
   end
 
-  @spec consolidate_silver_line_route_feature(atom) :: atom
-  def consolidate_silver_line_route_feature(:silver_line), do: :bus
-  def consolidate_silver_line_route_feature(icon_atom), do: icon_atom
```

Comment:
I'm trying to figure out, why would this function have been needed in the first place?

Reply:
OK I got curious and decided to look at our prior repo, dotcom-legacy (from before the project was open sourced). It turns out this function got added with [this pull request, "make silver line silver"](https://github.com/mbta/dotcom-legacy/pull/2788) :) 

Not sure if there's anything relevant to this PR but there's lotsa interesting discussion in that PR :D

Reply:
Ok so 2 things:

- If I understand that PR discussion correctly, they did want to replace the SL icon with the bus to avoid inconsistencies. I feel like keeping the function I deleted should be the way to go then.
BUT I also like seeing the gray icon in the stops page for stops that are "silver line stops". What are your thoughts on creating an "exception function" for those? I believe that'd be only for South Station and Airport but I'm not sure if that's the right approach.

- I also noticed there is an inconsistency for the Airport stop (which can be addressed separately):
<img width="412" alt="image" src="https://user-images.githubusercontent.com/61979382/107682122-ec805300-6c6d-11eb-94ca-cf3dbed513df.png">
<img width="254" alt="image" src="https://user-images.githubusercontent.com/61979382/107682133-ef7b4380-6c6d-11eb-84eb-f060ca1f2f91.png">

Reply:
@alicia-mbta It looks to me like your PR fixed that inconsistency, maybe I'm looking in the wrong place? https://green.dev.mbtace.com/stops/subway vs https://green.dev.mbtace.com/stops/place-aport?

Reply:
As to your first point, I like seeing the grey icon too... I'm not sure exactly which inconsistencies our predecessors were concerned about, but it seems like it might no longer matter. After all, we now definitely want to see the SL icon instead of the bus icon.

Honestly after poking around the site a bit I think deleting that function did the job great 👍 but we can let the product folks review and see if they find anything amiss too.

Reply:
Perfect, thanks a lot for the poking around and the feedback!